tests/permissions/conftest.py

```diff
@@ -0,0 +1,37 @@
+import shutil
+from subprocess import DEVNULL, run
+from time import perf_counter
+
+import lamindb_setup as ln_setup
+import pytest
+from lamin_utils import logger
+
+
+def pytest_sessionstart():
+    t_execute_start = perf_counter()
+
+    ln_setup.settings.auto_connect = True
+    # these are called in separate scrits because can't change connection
```

Comment:
Got it, you need to both access as an admin and then as a user to test.

Reply:
Yes, exactly.

Reply:
Typo in "scripts"